benchmarks/benchmarks/sparse_linalg_lobpcg.py

```diff
@@ -66,45 +22,119 @@ def __init__(self):
         self.time_mikota.__func__.setup = self.setup_mikota
 
         self.time_sakurai.__func__.params = list(self.params)
-        self.time_sakurai.__func__.params[0] = [50, 400]
+        self.time_sakurai.__func__.params[0] = [50, 100]
         self.time_sakurai.__func__.setup = self.setup_sakurai
 
+        self.time_sakuraii.__func__.params = list(self.params)
+        self.time_sakuraii.__func__.params[0] = [500, 1000]
+        self.time_sakuraii.__func__.setup = self.setup_sakuraii
+
+
     def setup_mikota(self, n, solver):
         self.shape = (n, n)
-        self.A, self.B = _mikota_pair(n)
-
-        if solver == 'eigh' and n >= 512:
-            # skip: slow, and not useful to benchmark
-            raise NotImplementedError()
+        mik = Mikota_pair(n)
+        mik_k = mik.k
+        mik_m = mik.m
+        self.Ac = mik_k
+        self.Aa = mik_k.toarray()
+        self.Bc = mik_m
+        self.Ba = mik_m.toarray()
+        self.Ab = mik_k.tobanded()
+        self.eigenvalues = mik.eigenvalues
+
+        # if solver == 'eigh' and n >= 512:
+        #     # skip: slow, and not useful to benchmark
+        #     raise NotImplementedError()
```

Comment:
is this meant to be commented out?

Reply:
Thanks! Good catch - forgot to put it back. n = 512 is not too slow, so now reads n > 512